docs/SystemDesign/1-TsFile/2-Format.md

```diff
@@ -236,7 +236,7 @@ ByteBuffer[] statistics = [
 |              Member Description              |            Member Type             |
 | :------------------------------------------: | :--------------------------------: |
 |              Number of devices               |                int                 |
-| Pairs of device name and deviceMetadataIndex | String, TsDeviceMetadataIndex pair |
+| Pairs of device name and metadataIndex | String, TsDeviceMetadataIndex pair |
```

Comment:
I think we should line up this column.

Reply:
I intend to organize and modify the design docs in later PR ... For we've really changed a lot : )